.travis.yml

```diff
@@ -12,8 +12,23 @@ before_install:
     - sudo apt-get -qq update
     - sudo apt-get -qq install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libselinux1-dev
     - sudo apt-get -qq remove libseccomp2
+    - sudo apt-get update
+    - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
```

Comment:
Is the version of docker that Travis supplies too old?

Reply:
I'm not sure, I just wanted to make sure I'd the latest greatest.  However Travis if failing to update duggan/bats for some reason.